addons/website_forum/static/src/xml/website_forum_templates.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website_forum.add_to_menu">
+        <div class="form-group mb0">
+            <label class="offset-md-3 col-md-9 text-left">
+                <input type="checkbox" required="required"/>
+            </label>
+        </div>
+    </t>
+    <t t-name="website_forum.select_forum_mode">
+        <div class="form-group row mb0">
+            <label class="col-md-3 col-form-label">Forum Mode:</label>
+            <div class="form-check form-check-inline">
```

Comment:
https://getbootstrap.com/docs/4.3/components/forms/#checkboxes-and-radios-1 -> we use the custom form elements not those one

Reply:
+ you can't mix cols and non-cols... you can see that the template is broken